### ISSUE TITLE
[1.x] VideoRoom tells IPv6 disabled while IPv6 working on the server

### ISSUE THREAD

**Issue by Jean-Daniel** [meetecho/janus-gateway#3011]
**What version of Janus is this happening on?**
version 1.0.3

**Have you tested a more recent version of Janus too?**
no. because there is no change in this part of the code since 1.0.3

**Additional context**

This is due to the  `ipv6_disabled` being set unconditionally to `TRUE` at line 3745 (this line should be removed).  

https://github.com/meetecho/janus-gateway/blob/2e2776c4f2f39d7622d287cda56563f2243ae404/src/plugins/janus_videoroom.c#L3740-L3750

**Comment by lminiero** [comment]
Good catch, that line is invalidating the checks we make before, I'll get rid of it :+1:

**Comment by lminiero** [comment]
I suspect it's a line I added for debugging that then forgot to remove... :hand_over_mouth:
